docs: Edge Cloud-init reference page (#1153)
spectrocloud/librarium: content/docs/04-clusters/03-edge.md

<WarningBox>

> Edge clusters are still in active development and are subject to changes. Review the Palette [release notes](/release-notes) for updates and changes.

Author: ```suggestion
Edge is still in active development and is subject to change. Review the Palette [release notes](/release-notes) for updates and changes.

```
